velox/docs/functions/spark/string.rst

```diff
@@ -10,7 +10,9 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
 .. spark:function:: chr(n) -> varchar
 
-    Returns the Unicode code point ``n`` as a single character string.
+    Returns a utf8 string of single ASCII character. The ASCII character has the binary
```

Comment:
Please, revise to take into account your comment below

`When the input is between 0x80 and 0xFF, the output will be a utf8 string.`

Reply:
@mbasmanova, Thanks for the comment! I copied the function description directly from spark doc. The description looks incorrect because 0x80 to 0xFF are not ascii codes. Maybe I should revise `ASCII character` to `unsigned character`.